backend/app/migration/versions/d6bfa3df105a_initial_migration.py

```
@@ -0,0 +1,124 @@
+"""initial migration
+
+Revision ID: d6bfa3df105a
+Revises: 
+Create Date: 2025-06-30 17:10:50.002601
+
+"""
+from typing import Sequence, Union
+
+from alembic import op
+import sqlalchemy as sa
+import sqlmodel
```

Ruff (0.11.9)

`sqlmodel` imported but unused

Remove unused import: `sqlmodel`

(F401)